Scripts/validate-3ds2-SDK-version.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
Should we add `validate-3ds2-SDK-version.sh` as a step on CI? At least after merge to `master` ?

Reply:
After merging is too late, this script will prevent your commit even locally, I wanted prevention as early as possible.